lib/src/element/setter.dart

```diff
@@ -0,0 +1,121 @@
+part of domino.element;
+
+/// Adds a style to an [Element] with [name] and [value]
+///
+/// Example:
+///     div(set: style('color', 'blue'))
+class StyleSetter implements Setter {
+  final String name;
```

Comment:
Do you think it is useful to expose the name and value here? If not, let's make them private (makes changing the interfaces easier = less breaking - if needed.)